collect/stressRelief.go

```diff
@@ -266,22 +268,24 @@ func (s *StressRelief) Recalc() {
 	s.stressLevel = uint(level)
 	s.reason = reason
 
-	if s.stressLevel >= s.deactivateLevel {
-		// we want make sure that stress relief is below the deactivate level for a minimum time after the last time we
-		// said it should be, so whenever it's above that value we push the time out
-		s.stayOnUntil = time.Now().Add(10 * time.Second)
-	}
-
 	switch s.mode {
 	case Never:
 		s.stressed = false
 	case Always:
 		s.stressed = true
 	case Monitor:
+		// If it's off, should we activate it?
 		if !s.stressed && s.stressLevel >= s.activateLevel {
 			s.stressed = true
 			s.Logger.Info().WithField("stress_level", s.stressLevel).WithField("reason", s.reason).Logf("StressRelief has been activated")
 		}
+		// We want make sure that stress relief is below the deactivate level
+		// for a minimum time after the last time we said it should be, so
+		// whenever it's above that value we push the time out.
+		if s.stressLevel >= s.deactivateLevel {
```

Comment:
Do ww need to push out the time whenever `level > deactivateLimit` or only when currently stressed and `level > deactivateLimit`?

Reply:
I added an extra test for isStressed.